### DISCUSSION TITLE
How do I pass a variable to tinyfilemanager?

### DISCUSSION THREAD

**Discussion by mjelkins** [prasathmani/tinyfilemanager#1209] [Q&A]
I want to use TFM to manage different websites on one server. I have my own front end and see TFM as some very useful software. I have some code on my side...
<input type="button" value="Manage Website"   onClick="window.location='tfm.php?aid=<?php print $aid?>'" title="Manage this Website">
Inside tinyfilemanager.php, near the top, I've added the line...
if(isset($_GET['aid'])) $aid=$_GET['aid']; else $aid=9;
Checking the URL - I never see "http://mystuff.com//tinyfilemanager.php?aid=23" Just  "http://mystuff.com//tinyfilemanager.php?p="
and never get the value I'm trying to send. Why?
I then plan to open a mysql database and read that record number to work out things like which website I should be viewing.

**Comment by mjelkins** [comment]
Sorry - the code on my side would be....
<input type="button" value="Manage Website" onClick="window.location='tinyfilemanager.php?aid=<?php print $aid?>'" title="Manage this Website">

**Comment by mjelkins** [comment]
Lets try again - code on my side is....
input type="button" value="Manage Website" onClick="window.location='tfm.php?aid=  ..php print $aid.. '" title="Manage this Website"
  Above enclosed in the appropriate angled brackets and "..." is the "brackets" around a php piece of code.

**Comment by prasathmani** [comment]
https://github.com/prasathmani/tinyfilemanager/issues/118#issuecomment-454507320
